helm/README.md

```diff
@@ -93,7 +88,7 @@ kubectl create namespace inference-ms
 A command like the one below will then use the latest chart version to install the version of NIM defined in the values file into the `inference-ms` namespace in your Kubernetes cluster. Modify it as required.
 
 ```bash
-helm --namespace inference-ms install my-nim nim-demo/nim-llm -f path/to/your/custom-values.yaml
+helm --namespace inference-ms install my-nim nim-llm/ -f path/to/your/custom-values.yaml
```

Comment:
In another PR let's create a `config` directory with an example values.yaml, .gitignore it, and assume users are using that custom-values.yaml file in all example commands.

Reply:
I'll put that up today